moztelemetry/filter_service.py

```diff
@@ -81,7 +82,7 @@ def get_daily_stats(self, from_date, to_date):
         for i in range(n_days + 1):
             submission_date = (from_date + relativedelta(days=i)).strftime("%Y%m%d")
             domain = self[submission_date[:-2]]
-            query = "select count(*) from {} where submissionDate='{}'".format(domain.name, submission_date)
+            query = "select count(*) from `{}` where submissionDate='{}'".format(domain.name, submission_date)
```

Comment:
Looks like the addition of [backticks](https://docs.aws.amazon.com/AmazonSimpleDB/latest/DeveloperGuide/QuotingRulesSelect.html) here is due to having dashes in some prefixes.